### PR TITLE
IOS-15408: (part 3) expose ongoing background session ids and some better naming and comments

### PR DESCRIPTION
**Pull request by thuyBox**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @thuyBox has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.h [hunk 1]
```diff
@@ -334,10 +334,15 @@
 - (BOOL)cacheBackgroundSessionIdFromExtension:(NSString *)backgroundSessionId error:(NSError **)outError;
 
 /**
- * Return all background session ids known to the app from extensions
- * which were cached previously using cacheBackgroundSessionIdFromExtension:error
+ * Return ongoing background session IDs
```
Comment (jlawton) [inline BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.h]: What is an "ongoing" session?

Comment (MashfiqueA) [reply BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.h]: Sessions that are currently active?

Comment (thuyBox) [reply BoxContentSDK/BoxContentSDK/BOXURLSessionCacheClient.h]: yes. Updated comments.

**Review by MashfiqueA — APPROVED** [review]
LGTM

**Pull request merged**